recipes/cppcheck/all/test_package/conanfile.py

```diff
@@ -7,3 +8,8 @@ class TestPackageConan(ConanFile):
     def test(self):
         if not tools.cross_building(self.settings):
             self.run("cppcheck --version", run_environment=True)
+            # On windows we need to explicitly use python to run the python script
```

Comment:
maybe we can add similar wrapper for Windows ("check-htmlreport.bat"), but current solution is sufficient as well